tests/osmosis_integration_test.rs

```diff
@@ -129,29 +115,6 @@ impl<'a, R: Runner<'a> + Querier, T: VaultToken + Clone> TokenRobot<'a, R, T> {
     }
 }
 
-struct RunnerMockDeps<'a, Q: Querier> {
-    pub storage: MockStorage,
-    pub api: MockApi,
-    pub querier: &'a Q,
-}
-
-impl<'a, Q: Querier> RunnerMockDeps<'a, Q> {
-    pub fn new(querier: &'a Q) -> Self {
-        Self {
-            storage: MockStorage::default(),
-            api: MockApi::default(),
-            querier,
-        }
-    }
-    pub fn as_ref(&'_ self) -> Deps<'_, Empty> {
-        Deps {
-            storage: &self.storage,
-            api: &self.api,
-            querier: QuerierWrapper::new(self.querier),
-        }
-    }
-}
-
```

Comment:
I don't know, which is why I just removed it. Probably we should create a contract to test this functionality instead of mocking deps. But that's a lot of work and probably we should just refactor the vault contract to not use cw-vault-token instead.